### PR TITLE
Small cleanup in docs conf file and added a favicon for the documentation pages

### PR DESCRIPTION
**Pull request by QuantumChemist**

### CODE REVIEW

**Comment by QuantumChemist** [comment]
@JaGeo do you like the idea with the favicon?

File: docs/conf.py [hunk 1]
```diff
@@ -152,18 +152,16 @@
...
     "repository_provider": "github",
     "repository_url": "https://github.com/JaGeo/autoplex",
     "use_repository_button": True,
     "use_issues_button": True,
 }
 
-
 # html_static_path = ["_static"]
 
-html_css_files = ['custom.css']
```
Comment (naik-aakash) [inline docs/conf.py:165]: I noticed you have move this to line above. But there is no `github.css` file in our `docs/_static` directory. Is that you wanted to add at some point and it got left out of commits somehow?

Comment (QuantumChemist) [reply docs/conf.py:165]: It was like this all the time, I simply removed the redundant line. Maybe I can remove github.css then?

**Comment by QuantumChemist** [comment]
It would look like this
image

**Comment by JaGeo** [comment]
@QuantumChemist sounds great!

**Review by JaGeo — APPROVED** [review]

**Pull request merged**